### ISSUE TITLE
How did you manage to save two models in to one (.ONNX)

### ISSUE THREAD

**Issue by ghost** [Dhyeythumar/PPO-algo-with-custom-Unity-environment#6]
Hi, i hope you are doing great.
According to your PPO code, you are saving the models in two separate .hdf5 extensions after training is done. As far as I know, if we would like to use the Unity editor instead of the binary file (.exe), we should use only one training file with .ONNX extension. how did you turn the models into an ONNX extension file?

**Comment by dhyeythumar** [comment]
Hi @johnprada , 
Yes, `.ONNX` is a different type of format to freeze a trained model as compared to `.hdf5`.

If I remember correctly, at the time of developing this code TensorFlow didn't had great support to save models in `.ONNX` hence I used the `.hdf5` format. And, yes `.hdf5` format is not supported by the Unity engine if you want to use it inside the Unity editor.

That's why in my case, I wrote the test.py script for testing the `.hdf5` model by directly using the `.exe` file.

You have 2 choices:
1. Reimplement the model save feature to save the model in `.ONNX` format.
2. Try using the ML-Agents trainer to train your RL agents.
